docs/rules/no-extra-parens.md

```diff
@@ -165,6 +166,17 @@ const Component = (<div />)
 const Component = (<div><p /></div>)
 ```
 
+### enforceForArrowConditionals
+
+Examples of **correct** code for this rule with the `"all"` and `{ "enforceForArrowConditionals": true }` options:
```

Comment:
This line is still incorrect -- it should say `{ "enforceForArrowConditionals": false }`.